pkg/cmd/admin/node/node_options.go

```diff
@@ -179,11 +179,31 @@ func (n *NodeOptions) GetPrinters(gvk unversioned.GroupVersionKind) (kubectl.Res
 		return nil, nil, err
 	}
 
-	printerWithHeaders, err := n.Printer(mapping, false, false, false, false, false, false, []string{})
+	printerWithHeaders, err := n.Printer(mapping, &kubectl.PrintOptions{
```

Comment:
@juanvallejo `false`, `""`, etc, are the *zero values* of their respective types. Normally we embrace using the zero value, without the need to explicitly initialize it, what would mean that there's no need to for the next several lines.

In code:

```go
// I'd expect Printer to understand a nil printOptions as "use the defaults".
printerWithHeaders, err := n.Printer(mapping, nil)

// Or, if for whatever reason we need to provide a non-nil value:
printerWithHeaders, err := n.Printer(mapping, &kubectl.PrintOptions{})

// Yet, if we want to set just some of the options (just an example):
printerWithHeaders, err := n.Printer(mapping, &kubectl.PrintOptions{Wide: true})
```